### ISSUE TITLE
UMD way of loading Raygun4JS fails to exception

### ISSUE THREAD

**Issue by Havunen** [MindscapeHQ/raygun4js#235]
We use raygun npm package to post errors.

Initializing the package fails to following exception (2.6.5):

```
Uncaught TypeError: window.raygunFactory is not a function
    at Object.constructNewRaygun (null:2764)
    at Object.init (null:2764)
    at onLoadHandler (null:2764)
```

Rollbacking to version 2.6.1 fixes the issue, I tested that 2.6.4 was also broken and 2.6.3 was not published to NPM.

**Comment by fundead** [comment]
Thanks for alerting us to this - cc @UberMouse , related to the Grunt dependency ordering perhaps?

**Comment by fundead** [comment]
I've identified and pushed a fix for this issue, and it's been released as v2.6.6 on the package managers including NPM, and will shortly also be available on the CDN.

Thanks again for the detailed report,

Callum
Raygun Limited
